.ivy/raise-version.sh

```diff
@@ -6,14 +6,6 @@ set -e
 # Update artifact versions in reactor POM and all modules POMs 
 mvn --batch-mode versions:set versions:commit -DnewVersion=${1} -DprocessAllModules
 
-# Update artifact versions in parent POMs 
-mvn -f compile-test/crm.maven/config/pom.xml --batch-mode versions:set versions:commit -DnewVersion=${1} -DprocessAllModules
-mvn -f deploy/application/maven/config/pom.xml  --batch-mode versions:set versions:commit -DnewVersion=${1} -DprocessAllModules
-mvn -f docker/maven/config/pom.xml --batch-mode versions:set versions:commit -DnewVersion=${1} -DprocessAllModules
-
-# Update artifacts parent version in reactor POM and all modules POMs
-sed -i -b -E  "s:(^\s\s\s\s<version>)(.*)(<\/version>):\1${1}\3:g" $(find . -name "pom.xml")
```

Comment:
this would raise our ivy project parent at a time when we don't want to raise it.